pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -59,35 +53,7 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 }
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
-	is := makeInjectionSet(binding, cred)
-
-	if err := b.injectSecret(is.secret); err != nil {
-		return err
-	}
-
-	return nil
-}
-
-func (b *k8sBindingInjector) Uninject(binding *servicecatalog.Binding) error {
-	return fmt.Errorf("Not implemented")
-}
-
-func (b *k8sBindingInjector) injectSecret(s *v1.Secret) error {
-	cmc := b.client.Core().Secrets("default")
-	_, err := cmc.Create(s)
-	return err
-}
-
-func makeInjectionSet(binding *servicecatalog.Binding, cred *brokerapi.Credential) *injectionSet {
-	secret := makeSecret(binding, cred)
-
-	return &injectionSet{
-		secret: secret,
-	}
-}
-
-func makeSecret(binding *servicecatalog.Binding, cred *brokerapi.Credential) *v1.Secret {
-	return &v1.Secret{
+	secret := &v1.Secret{
 		ObjectMeta: v1.ObjectMeta{
 			Name: binding.Name,
```

Comment:
`SecretRef` might be a poorly-chosen name.  IIRC this is a name we came up with on the last day of the first F2F meeting.  Perhaps `SecretName` is better?

Naming of the field aside, we need to be able to know the name of the objects that will be created based on the Binding spec, so that when the Binding includes the SIP spec, that the content of the SIP spec can address those objects by name.  Does that make sense?